src/Core/Constants.cs

```diff
@@ -81,7 +82,8 @@ public static List<string> GetAllKeys()
         return new Dictionary<string, string>()
         {
             { TrustedDeviceEncryption, "true" },
-            { Fido2VaultCredentials, "true" }
+            { Fido2VaultCredentials, "true" },
+            { GrantSaveOptimization, "false"}
```

Comment:
❌ This is a strange section of code, but is for overrides where we need to force non-default usage of feature flags for use cases like self-hosting. You won't need this change.